content/chainguard/chainguard-images/cve-risk.md

```diff
@@ -0,0 +1,62 @@
+
+
+## Recommended tools
+
+There are a number of tools available that allow you to scan your third party code for CVEs. At Chainguard, we use [Grype](https://github.com/anchore/grype) to scan our own Chainguard Images, as it's open-source and it can scan Software Bills of Materials (or SBOMs). Additionally, Grype biases towards false positives over false negatives. This suits our needs, as we'd rather look into a vulnerability in one of our images that turns out to be a false positive than overlook a real vulnerability that impacts our customers.
+
+Be aware that the results of a vulnerability scan can only tell you part of the story. Say, for example, that Project A has only a few dependencies while Project B has many. Logically, a vulnerability scan of Project A will likely return fewer results than one for Project B, but that doesn't mean that Project A is at any less of a risk. Although its results might show only a few CVEs, one of these vulnerabilities could still have a serious impact on Project A. Conversely, Project B's scan results might show a large number of CVEs, but most or all of these could be false positives. 
+
+One tool that's useful for automating updates for your dependencies is [Dependabot](https://docs.github.com/en/code-security/dependabot). Dependabot is a free tool offered by GitHub that sends alerts to repositories affected by new vulnerabilities and — if possible — raises a pull request to update the affected dependencies.
+
+It's important to treat every new dependency you introduce into an application with a degree of scrutiny. Although the needs of every project vary, there are a number of practices a team can implement to ensure that they only use dependencies that meet certain requirements. As an example, you could integrate a tool like [Scorecard](https://securityscorecards.dev/) into your organization's workflow.
+
+Scorecard is an automated tool from the Open Source Security Foundation that performs a variety of security checks on software, returning a score between 1 and 10 for each one. These scores can help you understand what you need to work on to improve your project's security, and can also help you assess the security of your dependencies. 
+
+[Chainguard Images](https://www.chainguard.dev/chainguard-images) are also helpful for minimizing your CVE risk. Many of Chainguard's images are distroless, meaning they contain only an application and its runtime dependencies. These images do not even contain a shell or package manager. Because Chainguard Images minimize the number of dependencies in this manner, they inherently contain few to zero CVEs. Also, Chainguard Images are rebuilt nightly to ensure images are completely up-to-date and contain all available security patches.
```

Comment:
```suggestion
[Chainguard Images](https://www.chainguard.dev/chainguard-images) can minimize your CVE risk. Many of Chainguard's images are distroless, meaning they contain only an application and its runtime dependencies. These images do not even contain a shell or package manager. Because Chainguard Images minimize the number of dependencies in this manner and thus reducing the potential attack surface, they inherently contain few to zero CVEs. Also, Chainguard Images are rebuilt nightly to ensure images are completely up-to-date and contain all available security patches, and the engineering team [often fixes vulnerabilities before they're detected](https://www.chainguard.dev/unchained/how-chainguard-fixes-vulnerabilities).
```